.agent/skills/manage-skills/SKILL.md

```
@@ -0,0 +1,336 @@
+| 스킬 | 갭 유형 | 상세 |
+|------|---------|------|
+| verify-api | 파일 누락 | `src/server/newHandler.ts`가 Related Files에 없음 |
+| verify-ui | 새 패턴 | 새 컴포넌트가 검사되지 않는 규칙을 사용 |
+| verify-test | 오래된 값 | 설정 파일의 테스트 러너 패턴이 변경됨 |
+```
+
+### Step 4: CREATE vs UPDATE 결정
+
+다음 결정 트리를 적용합니다:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)